builtin/game/fine_pointed.lua

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
why not rename it to exact poinjted position  ? it's more clear than fine pointed